crypto/ec/asm/ecp_nistz256-x86_64.pl

```diff
@@ -2601,7 +2627,22 @@
 	test	\$`1<<5`, %eax
 	jnz	.Lavx2_gather_w5
 ___
-$code.=<<___	if ($win64);
+$code.=<<___ if ($win64 && $use_tool_asm);
```

Comment:
This is suspicious.  Why shouldn't the callee saved register be saved if !$use_tool_asm ?
We still emit the AESNI instructions don't we?